source/_components/switch.ecoplug.markdown

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: EcoPlug/Wion Switch
+description: "Instructions on how to set up an EcoPlug-Compatible switch within Home Assistant."
+date: 2018-06-25 14:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Switch
+ha_release: 0.71
+ha_iot_class: "Local Polling"
+---
```

Comment:
Maybe add a logo?
That makes the documentation and the component a little more recognizable.